Windows/quickstart-cpp/CppHelloSpeech.cpp

```diff
@@ -11,7 +11,7 @@ using namespace Microsoft::CognitiveServices::Speech;
 
 void recognizeSpeech() {
     wstring subscriptionKey{ L"<Please replace with your subscription key>" };
-    wstring region{ L"<Please replace with your service region>" };
+    wstring region{ L"<Please replace with your service region (e.g., 'westus')>" };
```

Comment:
We need similar changes in our main repo, right?

Reply:
Yes.

Reply:
(these ones aren't in the main repo yet, but soon will be)